### PR TITLE
LPS-164527 [EVP] Create Organization form

### PR DESCRIPTION
**Pull request by ytuik**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request-organization.json [hunk 1]
```diff
@@ -8,14 +8,16 @@
...
+		"publish": {
+		},
 		"update": {
 		}
 	},
-	"active": true,
+	"active": false,
 	"label": {
-		"en_US": "EVP Organization"
+		"en_US": "EVP Request Organization"
```
Comment (silasbsi) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/request-organization.json:18]: I have a question at this point. Should it be "EVP Request Organization" or "EVP Organization Request"?

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some suggestions.

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Review by silasbsi — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/organization-request.json [hunk 2]
```diff
@@ -8,14 +8,16 @@
 		},
 		"permissions": {
 		},
+		"publish": {
+		},
 		"update": {
 		}
 	},
-	"active": true,
+	"active": false,
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/organization-request.json:16]: hey @ytuik @silasbsi. Why this prop is false?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/organization-request.json [hunk 3]
```diff
@@ -391,12 +393,12 @@
...
 	"parameterRequired": false,
 	"pluralLabel": {
-		"en_US": "EVP Organizations"
+		"en_US": "EVP Organization Requests"
 	},
 	"portlet": false,
 	"scope": "company",
 	"status": {
-		"code": 0,
+		"code": 2,
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/object-definitions/organization-request.json:401]: Same here, why this prop is 2?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/02_my_evp/01_organization-form/page-definition.json [hunk 4]
```diff
@@ -0,0 +1,573 @@
+{
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/02_my_evp/01_organization-form/page-definition.json:1]: Hi Bruno, could you add a button for accessing the form in MyEVP. The way it is now, I have no way to open it, just by editing the page.

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by juliocezarjc** [comment]
Just starting reviewing :)

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @ytuik

**Comment by JoMaAlves** [comment]
ci:test:sf

**Comment by ytuik** [comment]
ci:test

**Comment by ytuik** [comment]
ci:test:sf

**Comment by JoMaAlves** [comment]
Just starting reviewing :)

**Comment by JoMaAlves** [comment]
ci:test:relevant

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**